hack/bin/deps.bzl

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
```

Comment:
[buildifier](https://github.com/bazelbuild/buildtools) did this 😅

Reply:
alphabetic ordering 😅